packages/expo-cli/src/commands/upgrade.ts

```diff
@@ -100,6 +100,20 @@ async function getUpdatedDependenciesAsync(
   return result;
 }
 
+async function upgradeAppJson(projectRoot: string, targetSdkVersionString: string) {
+  let { exp } = await ConfigUtils.readConfigJsonAsync(projectRoot);
+  exp.sdkVersion = targetSdkVersionString;
+  switch (targetSdkVersionString) {
+    case '37.0.0':
+      if (exp.androidNavigationBar?.visible === false) {
```

Comment:
we should log something to users to let them know what change we are making if we remove the `androidNavigationBar.visible` key or change it